src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -22,9 +22,8 @@ import { ONE_HUNDRED_PERCENT, ZERO_PERCENT } from 'constants/misc'
 import { PERCENTAGE_PRECISION } from 'constants/index'
 
 enum ErrorMsgs {
-  Initial = 'Insufficient balance to cover the selected investment amount. Either modify the investment amount or unselect the investment option to move forward',
-  Balance = 'Insufficient balance to cover the selected investment amount, please modify the selected amount to move forward',
-  MaxCost = 'Selected amount is higher than available investment amount, please modify the input amount to move forward',
+  InsufficientBalance = 'Insufficient balance to cover the investment',
```

Comment:
`the investment` sounds weird, I would say:
```suggestion
  InsufficientBalance = 'Insufficient balance to cover investment amount',
```